File: esercizi_java/test/compito_python/auto.py
from veicolo import veicolo

class auto(veicolo):
    def __init__(self, cod, targa, modello, marca, cost, cilindrata, bagaglio):
        super().__init__(cod, targa, modello, marca, cost)
        self.cilindrata = cilindrata
        self.bagaglio = bagaglio

    def toString(self):
        return "automobile" + "\t" + super().toString() + str(self.cilindrata) + "\t" + str(self.bagaglio) + "\t-\t-\t-\t"

File: esercizi_java/NetBeansProjects/ciaone/src/gestione/auto.java
package gestione;

/**
 *
 * @author tobiasacchetto
 */
public class auto extends veicolo{
    private int cilindrata, bagagliaio;
    
    public auto(int cod,String targa,String modello,String marca,double cost,int cilindrata,int bagagliaio){
        super(cod,targa,modello,marca,cost);
        this.cilindrata = cilindrata;
        this.bagagliaio = bagagliaio;
    }
    
    @Override
    public String toString(){
        return "automobile"+ "\t" + super.toString() +cilindrata + "\t"+ bagagliaio+ "\t-"+ "\t-"+ "\t-";
    }
    
    
}
